EduVPN-redesign/Networking/DiscoveryDataFetcher.swift

```diff
@@ -9,11 +9,20 @@ import Foundation
 import Moya
 import PromiseKit
 
-enum DiscoveryDataFetcherError: LocalizedError {
+enum DiscoveryDataFetcherError: Error {
     case dataCouldNotBeVerified
     case dataNotFoundInCache
 }
 
+extension DiscoveryDataFetcherError: AppError {
+    var summary: String {
+        switch self {
+        case .dataCouldNotBeVerified: return "Discovery data could not be verified"
+        case .dataNotFoundInCache: return "Discovery data not found in cache"
```

Comment:
Not sure if these are shown to user, if yes, make localizable.

Reply:
These won't get shown to the user.